pkg/minikube/cruntime/cruntime.go

```diff
@@ -55,6 +55,10 @@ func ValidRuntimes() []string {
 // CommandRunner is the subset of command.Runner this package consumes
 type CommandRunner interface {
 	RunCmd(cmd *exec.Cmd) (*command.RunResult, error)
```

Comment:
May be add a comment here to describe how RunCmd is different from Start/WaitCmd, and when it should be used

Reply:
Thanks for bringing this up, I added additional comments that hopefully make it more clear, let me know what you think.